docs/tutorials/bpy_module.md

```diff
@@ -0,0 +1,43 @@
+# Using `bpy` module from pip
+
+BlenderProc supports using `bpy` module installed from pip. This behavior is controlled using the `USE_EXTERNAL_BPY_MODULE=1` environment variable.
+
+## Setup
+
+1. Set `USE_EXTERNAL_BPY_MODULE=1` in your environment
+2. `pip install bpy==4.2.0`
+3. Install [required packages](#required-dependencies)
+4. After `import blenderproc` call `blenderproc.init()`
+5. Run your script using `python my_script.py` or `blenderproc run my_script.py`
+
+## Limitations
+- BlenderProc Blender dependencies have to be installed manually, see [required dependencies](#required-dependencies).
+- **Requires Python** version **3.11** - this is bound to the version `bpy` and Blender uses, it is the Python version suggested by [VFX Reference Platform](https://vfxplatform.com/).
+- All [limitations](https://docs.blender.org/api/current/info_advanced_blender_as_bpy.html#limitations) of the `bpy` module itself.
+- `blenderproc debug` command is not available, as `bpy` module does not support user interface.
+- `blenderproc pip` commands are not available. Use the `pip` commands in your environment instead.
+
+## Required dependencies
+Following dependencies have to be installed in your environment to allow using BlenderProc with Python `bpy` module. 
+
+```
+wheel
+pyyaml==6.0.1
+imageio==2.34.1
+gitpython==3.1.43
+scikit-image==0.23.2
+pypng==0.20220715.0
+scipy==1.13.1
+matplotlib==3.9.0
+pytz==2024.1
+h5py==3.11.0
+Pillow==10.3.0
+opencv-contrib-python==4.10.*
+scikit-learn==1.5.0
+python-dateutil==2.9.0.post0
+rich==13.7.1
+trimesh==4.4.0
+pyrender==0.1.45
```

Comment:
Agree, also note for myself - PyQT5 might be required (at least on Win) to prevent

```
c:\users\gripe\polygoniq_projects\blenderproc\blenderproc\scripts\visHdf5Files.py:260: UserWarning: FigureCanvasAgg is non-interactive, and thus cannot be shown
  plt.show()
```

Reply:
Fixed, the PyQT5 didn't solve my problem, it might be related to my venv setup, as in both modes now `vis` reports this error. Keeping the thread open, will investigate.

Can you please try using the `vis` command in your env, thanks?